corehq/apps/cleanup/management/commands/populate_sql_model_from_couch_model.py

```diff
@@ -51,6 +51,15 @@ def diff_couch_and_sql(cls, couch, sql):
         """
         raise NotImplementedError()
 
+    @classmethod
+    def diff_attr(cls, name, doc, obj, wrap=None):
+        couch = doc.get(name, None)
+        sql = getattr(obj, name, None)
+        if wrap:
+            couch = wrap(couch)
+        if couch != sql:
+            return f"{name}: couch value '{couch}' != sql value '{sql}'"
```

Comment:
nit: formatting with `!r` would make it possible to differentiate between an int and a string containing the same number.
```suggestion
            return f"{name}: couch value {couch!r} != sql value {sql!r}"
```

Reply:
Thank you, I didn't know about `!r`